client/src/components/new-request/main-container.vue

```diff
@@ -57,8 +57,18 @@ export default class MainContainer extends Mixins(SessionTimerMixin, DisplayedCo
     return newReqModule.editMode
   }
 
-  get isExistingRequestDisplay (): boolean {
-    return (newReqModule.displayedComponent === 'ExistingRequestDisplay')
+  get showExit (): boolean {
+    return [2, 3].includes(newReqModule.submissionTabNumber) ||
+      newReqModule.displayedComponent === 'ExistingRequestDisplay'
+  }
+
+  backToSearch () {
+    if ([2, 3].includes(newReqModule.submissionTabNumber) &&
+      newReqModule.displayedComponent !== 'ExistingRequestDisplay') {
+      newReqModule.mutateExitModalVisible(true)
+    } else {
+      this.cancelAndResetState()
+    }
```

Comment:
OK I'm being picky here, but I think it would more understandable if this was a switch-case statement.

Reply:
It may not be a good idea since we have multiple condition to decide